src/Confluent.SchemaRegistry/Confluent.SchemaRegistry.csproj

```diff
@@ -26,4 +26,8 @@
     <PackageReference Include="System.Net.Http" Version="4.3.0" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net452' ">
```

Comment:
do you have an opinion on dropping `net452` support in schema registry entirely?

the official apache avro library, which we're migrating to away from our fork, targets `netstandard2.0` only. We debated this and plan to make the switch in the next release (v1.4) rather holding off and doing a major version bump (v2.0) since we think the negative impact will be relatively low.

this change has already been made on the 1.4.x branch. you might like to target that, though we won't incorporate your PR in 1.4, so it's probably better to leave it hanging for a bit.

Reply:
I think dropping <code>net452</code> is a good way to go,  in schema registry 

As soon as you release version 1.4, I will change the implementation for dropping support to it